packages/jobs/src/adapters/BaseAdapter/__tests__/BaseAdapter.test.ts

```diff
@@ -0,0 +1,41 @@
+import { describe, expect, vi, it } from 'vitest'
+
+import { BaseAdapter } from '../BaseAdapter'
+import type { BaseAdapterOptions } from '../BaseAdapter'
+
+const mockLogger = {
```

Comment:
You did define a mock logger in a little mocks file if you want to use that here?